core/src/test/java/greencity/service/impl/EcoNewsServiceImplTest.java

```diff
@@ -104,8 +93,32 @@ public void save() throws MalformedURLException {
         addEcoNewsDtoResponse.setTitle("test title");
     }
 
+    @Test
+    void saveWithExistedImage() throws IOException {
+        MultipartFile image = ModelUtils.getFile();
+        String imageToEncode = Base64.getEncoder().encodeToString(image.getBytes());
+        addEcoNewsDtoRequest.setImage(imageToEncode);
```

Comment:
Could you please use blank line to separate into 3 blocks:
- GIVEN (initialization)
- WHEN (execution)
- THEN (assertions)